src/wazuh_modules/vulnerability_scanner/tests/unit/exclusionScanner_test.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Wazuh Vulnerability Scanner - Unit Tests
+ * Copyright (C) 2015, Wazuh Inc.
+ * September 21, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "exclusionScanner_test.hpp"
+#include "scanOrchestrator/exclusionScanner.hpp"
+#include "scanOrchestrator/scanContext.hpp"
+#include <memory>
```

Comment:
Missing header (for `std::copy`)
```suggestion
#include <algorithm>
#include <memory>
```

Reply:
There are several cases like this across all test files.